src/shared/utils/__tests__/utils.test.ts

```diff
@@ -4,60 +4,108 @@ import { SchemaId, WrappedDocument } from "../../../shared/@types/document";
 import * as v2 from "../../../__generated__/schema.2.0";
 import * as v3 from "../../../__generated__/schema.3.0";
 
+const baseIssuerObjectV2: v2.Issuer = {
+  name: "John",
+  identityProof: {
+    type: v2.IdentityProofType.DNSTxt,
+    location: "tradetrust.io",
```

Comment:
```suggestion
    location: "https://example.com",
```
explicitly as a sample?